examples/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  sc:
+    image: infinyon/fluvio:stable
+    container_name: sc
+    hostname: sc
+    ports:
+      - "9103:9003"
+    environment:
+      - RUST_LOG=info
+    command: "./fluvio-run sc --local /fluvio/metadata"
```

Comment:
I think we lost the volumes mappings from the original docker-compose.  
volumes:
 - ./metadata:/fluvio/metadata

Reply:
Yeah! I did this on purpose to avoid creating directories in the user's PWD.
Volumes are being introduced as Docker Volumes here:

```yml
volumes:
  fluvio-metadata:
  fluvio-data:
```

It still creates these volumes but Docker manage them. Do you rather leave them in the user side for easy inspection?

Reply:
That takes care of the persistency of them. Can the user still inspect and delete them?  Where are they created?

Reply:
A user could find existing volumes with `docker volume list` and then remove them
using: `docker volume rm`.

```
docker volume list
DRIVER    VOLUME NAME
local     buildx_buildkit_frosty_bartik0_state
local     docker-compose_fluvio-data
local     fluvio-docker_fluvio-data
local     fluvio-docker_fluvio-metadata
```

But accessing them is more obscure. It would have to be done by ssh-ing into the container.

Reply:
If inspecting these directories is important for example purposes, I would just
use local directories as we were using.

They are automatically created while docker spins up containers.

Reply:
Moved into local volumes for convenience. [0a92722](https://github.com/infinyon/fluvio/pull/3910/commits/0a92722046fda9611d80b7c23477a30cdfcf317a)